### ISSUE TITLE
no-trailing-punctuation config option is ignored

### ISSUE THREAD

**Issue by ksnyde** [DavidAnson/vscode-markdownlint#68]
in my `.markdownlint.yaml` file (in repo root) I have several rules (see below) which all work but I wanted to modify `no-trailing-punctuation` to allow both "!" and "?"

```yaml
no-duplicate-header: false
line-length: false
no-inline-html: false
no-space-in-links: false
no-trailing-punctuation: ".,;:"
```

But as many combinations as I try I can't get this configuration adjustment to take hold. Also of note, as this extension is really just living off thew shoulder of the underlying `markdownlint` I have added [this issue](https://github.com/DavidAnson/markdownlint/issues/174) there as well because I'm not sure how to distinguish between the two.

**Comment by DavidAnson** [comment]
It’s a YAML syntax issue, I have more info here: 
https://github.com/DavidAnson/markdownlint/issues/174
